packages/eslint-scope/README.md

```diff
@@ -0,0 +1,110 @@
+[![npm version](https://img.shields.io/npm/v/eslint-scope.svg)](https://www.npmjs.com/package/eslint-scope)
+[![Downloads](https://img.shields.io/npm/dm/eslint-scope.svg)](https://www.npmjs.com/package/eslint-scope)
+[![Build Status](https://github.com/eslint/eslint-scope/workflows/CI/badge.svg)](https://github.com/eslint/eslint-scope/actions)
```

Comment:
need to update the link after the repo being renamed.
```suggestion
[![Build Status](https://github.com/eslint/eslint-scope/workflows/CI/badge.svg)](https://github.com/eslint/js/actions)
```

Reply:
Let's update it only after the repo is renamed,